### PR TITLE
Publish core project

### PR DESCRIPTION
**Pull request by OlivierChirouze**

### CODE REVIEW

File: paf-mvp-core-js/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        if (DEV) {
+          return [];
+        } else {
+          return [
+            terser() // minify js output
+          ];
+        }
+      })()
+    ],
+    // Ignore all npm dependencies
```
Comment (RomainLofaso) [inline paf-mvp-core-js/rollup.config.js:62]: The reason for those ignored dependencies isn't clear to me. Can you add a sentence about why we ignore those dependencies, please?

File: paf-mvp-core-js/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ...(() => {
+        if (DEV) {
+          return [];
+        } else {
+          return [
+            terser() // minify js output
+          ];
+        }
+      })()
+    ],
```
Comment (RomainLofaso) [inline paf-mvp-core-js/rollup.config.js:61]: OPT: Do you think that this `if (DEV)` can be encapsulated in a function to add semantics? `minimifyIfProduction()`?

Side note, we use the spread operator here with an empty array or an array containing one element. Does it make sense to return a simple value without a spread? Or is the empty value annoying for that?

File: paf-mvp-core-js/rollup.config.js (hunk 1 above)
Comment (OlivierChirouze) [reply paf-mvp-core-js/rollup.config.js]: added a comment

File: paf-mvp-core-js/rollup.config.js (hunk 2 above)
Comment (OlivierChirouze) [reply paf-mvp-core-js/rollup.config.js]: I haven't created a function but I think I simplified

**Review by RomainLofaso — APPROVED** [review]

**Pull request merged**